src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -66,23 +66,32 @@
                                                         (nav/go-to-path :method id)
                                                         (scroll-to-top))}])}])
         (when owner?
-          [:div {}
-            [comps/SidebarButton
-             {:style :light :color :button-primary
-              :text "Permissions..." :icon :settings :margin :bottom
-              :onClick #(modal/push-modal
-                         [mca/AgoraPermsEditor
-                          {:save-endpoint (endpoints/persist-agora-method-acl entity)
-                           :load-endpoint (let [{:keys [name namespace snapshotId]} entity]
-                                            (endpoints/get-agora-method-acl namespace name snapshotId config?))
-                           :entityType (:entityType entity)
-                           :entityName (mca/get-ordered-name entity)
-                           :title (str (:entityType entity) " " (mca/get-ordered-name entity))}])}]
-            [comps/SidebarButton
-             {:style :light :color :exception-state
-              :text "Redact" :icon :delete :margin :bottom
-              :onClick #(modal/push-modal [Redactor {:entity entity :config? config?
-                                                 :on-delete (:on-delete props)}])}]])])
+          (list
```

Comment:
ahhh ... `list`. That extra `[:div {}` up there was my code because I couldn't get the multiple buttons working without it. Thank you!